.github/CODEOWNERS

```diff
@@ -83,10 +87,7 @@
 
 # Multiple Owners
 
-/code/__DEFINES/components.dm @Cyberboss @ninjanomnom
 /code/controllers/subsystem/air.dm @duncathan @MrStonedOne
-/code/datums/components/ @Cyberboss @ninjanomnom
```

Comment:
I'm assuming you don't care about this notification anymore since I don't think you've taken an interest in any of them for a long time @Cyberboss

Reply:
Too many things on it now to watch